src/backend/effects/builtin/chat.js

```diff
@@ -19,6 +19,10 @@ const effect = {
         <textarea ng-model="effect.message" class="form-control" name="text" placeholder="Enter message" rows="4" cols="40" replace-variables></textarea>
         <div style="color: #fb7373;" ng-if="effect.message && effect.message.length > 500">Chat messages cannot be longer than 500 characters. This message will get automatically chunked into multiple messages if it's too long after all replace variables have been populated.</div>
         <div style="display: flex; flex-direction: row; width: 100%; height: 36px; margin: 10px 0 10px; align-items: center;">
+            <label class="control-fb control--checkbox" style="margin: 0px 15px 0px 0px"> Use "/me"<tooltip text="'Applies Italics to your Chat Message or your Chat Color if used in a Whisper'"></tooltip>
+                <input type="checkbox" ng-model="effect.me">
+                <div class="control__indicator"></div>
+            </label>
```

Comment:
I know you are just mirroring the other checkboxes here, but worth mentioning that we now have a proper firebot checkbox component (created more recently) that we've been slowly refactoring to. Wouldn't hurt to use that here so we aren't adding additional instances of the "old way". Here's how to use it:
```html
<firebot-checkbox 
    label="Use \"/me\"" 
    tooltip="Applies Italics to your Chat Message or your Chat Color if used in a Whisper" 
    model="effect.me"
    style="margin: 0px 15px 0px 0px"
 />
```

Reply:
Perfect, thank you! I've gone through and updated the other checkboxes and the input box to use these Firebot Components in my latest commit.